src/Screens/Gameplay/Components/Map/FocusedMap/FocusedMap.css

```diff
@@ -4,8 +4,11 @@
 	left: calc(50% - calc(calc(0.5 * var(--NumOfColumns)) * var(--height-unit)));
 	/* will need to connect NumOfCoulmns logic via styled components later on. */
 	bottom: 0;
+	background-image: url('Assets/FocusedMap/focusedMapBorder.png');
+	background-size: cover;
```

Comment:
about this image -

_**1.**_ it can't exist in the same folder as all the tiles, it becomes almost invisible.
if it survives this review after the second note - please move all tile assets to a sub-folder named `Tiles` inside `Assets/FocusedMap`

_**2.**_ it doesn't fit well at the edges, sorry.
why not just make a polygon? it'll be exact, easily changeable, and will fit existing practices.